packages/react-composites/tests/browser/common/testapp.webpack.config.js

```diff
@@ -46,12 +47,21 @@ module.exports = (appDir, babelConfig) => {
           use: ['style-loader', 'css-loader']
         },
         {
-          test: /\.svg/,
+          test: /\.(eot|svg|ttf|woff|woff2)$/,
           type: 'asset/inline'
         }
       ]
     },
-    plugins: [new HtmlWebpackPlugin({ template: path.join(__dirname, 'index.html') })],
+    plugins: [
+      new HtmlWebpackPlugin({ template: path.join(__dirname, 'index.html') }),
+      new CopyWebpackPlugin({
+        patterns: [
+          { from: path.join(__dirname, 'fonts', 'segoeui-bold.woff2'), to: 'fonts' },
```

Comment:
Can we not just copy everything inside `__dirname/'fonts'` (why list the files?)

Reply:
Purely because I weirdly kept getting:
![image](https://user-images.githubusercontent.com/2684369/150223145-dbcdc306-d38c-431e-bda7-43243714eade.png)
For all the variations of:
```
{ from: path.join(__dirname, 'fonts', '*.woff2'), to: 'fonts' }
```
Do you know what it should be? Acoording to the documentation that wildcard should work^

Reply:
Not in the mood to fight a webpack plugin this afternoon. Will trust that you tried "long 'nuff"